scio-test/src/test/scala/com/spotify/scio/values/PairSCollectionFunctionsTest.scala

```diff
@@ -557,6 +557,18 @@ class PairSCollectionFunctionsTest extends PipelineSpec {
     }
   }
 
+  it should "support topByKey() with Iterable" in {
+    runWithContext { sc =>
+      val p = sc.parallelize(Seq(("a", 1), ("b", 11), ("b", 12), ("c", 21), ("c", 22), ("c", 23)))
+      val r1 = p.topByKey(1)
+      val r2 = p.topByKey(1, Ordering.by(-_))
+      r1 should
+        containInAnyOrder(Seq(("a", Iterable(1)), ("b", Iterable(12)), ("c", Iterable(23))))
```

Comment:
in addition to new test ... you could potentially deprecate the `iterable` utility and update the tests that use it.

Reply:
yeah, sure.. I am just compiling and running tests in the cloud to save battery! I'll update the pr soon..